oap-server/server-starter/src/main/resources/lal/mysql-slowsql.yaml

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+rules:
+  - name: default
+    dsl: |
+      filter {
+        json{
+        }
+        slowSql {
```

Comment:
How we prevent other logs into this log? If we don't have any pre conditions?

Reply:
I think it's more flexible to leave the settings to users.
such as following:
```
rules:
  - name: mysql-slowsql
    dsl: |
      filter {
        json{
        }
        if(parsed.isSlowSql){
          slowSql {
                    serviceName parsed.service as String
                    id parsed.id as String
                    statement parsed.statement as String
                    latency parsed.query_time as Long
                    layer parsed.layer as String
                    timeBucket parsed.time as Long
                  }
        }
        sink{
        }
      }
```

Reply:
How parsed.isSlowSql is to be true, is the key question. I don't think change protocol to add  field is good idea.

In showcase, when you finished this and added PostgreSQL slow SQLs, you would face how to distinguish them.

Reply:
It is not a question.
`parsed.isSlowSql` should set by log collector tool (such as fluent-bit), don't need to change protocol. Such as following
```
[{"date":1662478799.208541,"body":{"json":{"json":"{\"time\":\"20220906153959\",\"isSlowSQL\":\"true\",,\"id\":\"cb92c1a5b-2691e-fb2f-457a-9c72a392d9ed\",\"service\":\"root[root]@[175.172.37.237]\",\"statement\":\"SET timestamp=1662478799;select sleep(2);\",\"layer\":\"MYSQL\",\"query_time\":2000}"}},"service":"root[root]@[175.172.37.237]"}]
```
Just set the JSON to `log.body.json.json` and the parse.isSlowSql will have value.

I distinguish PostgreSQL and MySQL by Layer. Just like the JSON before.

Reply:
@kezhenxu94 Is this kind of value injection expected from design perspective? This seems like a dynamic injecting.
This may not a security issue, but  it is a protocol hijacking.

@yswdqz I prefer this flag is a tag in tags.

Reply:
> @kezhenxu94 Is this kind of value injection expected from design perspective? This seems like a dynamic injecting. This may not a security issue, but it is a protocol hijacking.
> 
> @yswdqz I prefer this flag is a tag in tags.

I get it.